pkg/oc/cli/cmd/set/buildhook.go

```diff
@@ -235,39 +219,44 @@ func (o *BuildHookOptions) Run() error {
 		return fmt.Errorf("%s/%s is not a build config", infos[0].Mapping.Resource, infos[0].Name)
 	}
 
-	if len(o.Output) > 0 || o.Local || kcmdutil.GetDryRunFlag(o.Cmd) {
-		return o.PrintObject(infos)
-	}
-
-	failed := false
+	allErrs := []error{}
 	for _, patch := range patches {
 		info := patch.Info
+		name := info.Name
+		if info.Mapping != nil {
```

Comment:
This fixes the panic we were seeing with --local, right?

Reply:
Yup.

Reply:
Per @liggitt's comment [here](https://github.com/kubernetes/kubernetes/pull/65636#discussion_r199308079), I am wondering if we would would want to do the same in Origin? That is, default to showing lower-case kind.group (with `.group` being included when non-empty) when `info.Mapping` is nil?

Reply:
I've created `GetObjectName` which I'm guessing you can just add upstream, it does exactly what Jordan asked for.